dir.props

```diff
@@ -25,7 +25,7 @@
    <!-- Common repo directories -->
   <PropertyGroup>
     <BuildToolsVersion>1.0.25-prerelease-00115</BuildToolsVersion>
-    <CompilerToolsVersion>1.0.0</CompilerToolsVersion>
+    <CompilerToolsVersion>1.1.0-rc1</CompilerToolsVersion>
```

Comment:
Note that this is not too useful -- only controls the target files copied. Binaries are actually copied based on the setting in targets\runtime.project.json, which is 1.1.0-beta-1, and should be OK. Updating to a higher version is problematic since it would require updates to other packages. Will do in separate PR.